src/bezpath.rs

```diff
@@ -15,6 +17,9 @@ use crate::{
     ParamCurveExtrema, ParamCurveNearest, Point, QuadBez, Rect, Shape, TranslateScale, Vec2,
 };
 
+#[allow(unused_imports)]
+use crate::common::FloatFuncs;
```

Comment:
If this is only needed for a feature, I would prefer it to just be `cfg`'d on that feature, instead of just always allowed.